### PR TITLE
Remove unexpected char in email

### LINKED ISSUE #399: FFaker::Internet email, safe_email and free_email generating invalid email addresses
In FFaker::Internet methods email, safe_email and free_email are generating email addresses with underscores which are not valid.

### PR DESCRIPTION
**Pull request by marocchino**
close #399

### CODE REVIEW

**Comment by davidhrbac** [comment]
https://stackoverflow.com/questions/2049502/what-characters-are-allowed-in-an-email-address

File: lib/ffaker/internet.rb [hunk 1]
```diff
@@ -39,17 +39,15 @@ def user_name(name = nil)
...
 
     def domain_name
       [domain_word, domain_suffix].join('.')
     end
 
     def domain_word
-      dw = Company.name.split(' ').first
-      dw.gsub!(/\W/, '')
-      dw.downcase
+      sanitize(Company.name)
```
Comment (Volosh1n) [inline lib/ffaker/internet.rb]: I'd say this method be better returning the same value
```ruby
sanitize(Company.name.split(' ').first)
```

Comment (Volosh1n) [inline lib/ffaker/internet.rb]:
```suggestion
sanitize(Company.name.split(' ').first)
```

Comment (Volosh1n) [inline lib/ffaker/internet.rb]: I'd say this method should return same value, but sanitized.
```ruby
sanitize(Company.name.split(' ').first)
```

Comment (marocchino) [reply lib/ffaker/internet.rb]: 5e4f8dc

**Pull request merged**